### ISSUE TITLE
Autocomplete with custom render doesn't accept click

### ISSUE THREAD

**Issue by mhuffman** [alexgorbatchev/jquery-textext#72]
If you custom render items on autocomplete dropdown, using div or span, you can select the items with up/down arrows and press enter, however you can not select with mouse click.

This is the same on latest chrome and firefox.

You can see an example you your own example here:

[http://textextjs.com/manual/examples/autocomplete-with-custom-render.html](http://textextjs.com/manual/examples/autocomplete-with-custom-render.html)

Just press the letter 'P' to get multiple items in the dropdown and try to select using mouse click.
You will notice that you can select using arrow keys and enter.

**Comment by SharpNoiZy** [comment]
Yes, I can confirm this bug, but I think that alexgorbatchev is not longer working on this project ;(
So many issus but no commits from him for weeks now.

**Comment by JlazHH** [comment]
I discovered the offending code in textext.plugin.autocomplete.js. In the onClick function, there is the following line:

```
if(target.is(CSS_DOT_SUGGESTION) || target.is(CSS_DOT_LABEL))
            self.trigger('enterKeyPress');
```

This explicitly filters out custom renderings, since the custom rendered elements will not have the .text-suggestion or .text-label classes referred to here.

You can fix this by changing the line to something like:

```
if (target.is(CSS_DOT_SUGGESTION) || target.is(CSS_DOT_LABEL) || target.parent(CSS_DOT_SUGGESTION + "," + CSS_DOT_LABEL).length>0)
            self.trigger('enterKeyPress');
```

Or if you are using full minified code, search for the second reference to onClick and replace:

```
(d.is(h)||d.is(j))
```

with

```
(d.is(h)||d.is(j)||d.parent(h+','+j).length>0)
```

When I find time I will submit a pull request

**Comment by wrygiel** [comment]
@JlazHH, thanks!

Also, here is a different approach (if you don't want to modify the code yet):

```
render: function(suggestion) {
    var $span = $('<span>').html("html <b>with</b> <i>some</i> <u>formatting</u>");
    $span.find("*").addBack().addClass("text-label");
    return $span;
}
```

**Comment by miravukojevic** [comment]
I found a solution with simple css hack.
Just put on span smaller z-index (for example -1).
